cli/image-scanning-by-using-the-roxctl-cli.adoc

```diff
@@ -0,0 +1,68 @@
+[id="image-scanning-by-using-the-roxctl-cli"]
+:_mod-docs-content-type: ASSEMBLY
+= Image scanning by using the roxctl CLI
+include::modules/common-attributes.adoc[]
+:context: image-scanning-by-using-the-roxctl-cli
+
+toc::[]
+
+[role="_abstract"]
+You can scan images stored in image registries, including cluster local registries such as the {ocp} integrated image registry by using the `roxctl` CLI.
+
+[id="scanning-images-in-a-remote-cluster_{context}"]
+== Scanning images using a remote cluster
+
+By specifying the appropriate cluster using the delegated scanning configuration or the cluster parameter described in the following procedure, you can scan images from cluster local registries by using a remote cluster.
```

Comment:
```suggestion
By specifying the appropriate cluster that uses the delegated scanning configuration or the cluster parameter described in the following procedure, you can scan images from cluster local registries by using a remote cluster.
```
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#u

Reply:
@dcaravel This suggestion sgtm; hope that the technical meaning does not change by incorporating this suggestion.

Reply:
I believe it does change the technical meaning - the `cluster` is specified `in` the `delegated scanning config` - the change suggests that a cluster is being specified to `use` the `delegated scanning config`, which is not accurate.

Perhaps:

```suggestion
By specifying the appropriate cluster in the delegated scanning configuration or via the cluster parameter described in the following procedure, you can scan images from cluster local registries by using a remote cluster.
```